projects/36/bhxiv/paper.md

```diff
@@ -312,7 +312,7 @@ To sum up, we achieved our overarching goal of mapping GA4GH Phenopackets and OH
 4. set up GA4GH Beacon for Phenopackets standard.
 
 ## Contributions
-NQR, JJ, PR, VS and KP conceived the initial project. DW, AS, TC and NQR did the OMOP to Phenopackets schema mapping. GD and PR did the generation pipeline of the synthetic OMOP database. AS created the SQL script and jointly with JJ they created the OMOP exporter code. RK, VT, JJ, VS and NQR did the modelling of semantic Phenopackets for Semantic Web based federated health infrastructures. PA and RK did the OMOP to semantic Phenopackets pipeline. RK, CF, RG, MK, LJC and NQR did the FAIR Machine Learning approaches and modelling discussion. NQR and TC drafted the introduction and the discussion sections, and NQR polished a final version of the manuscript. All authors reviewed and approved the final version of the manuscript.
+NQR, JJ, PR, VS and KP conceived the initial project. DW, AS, TC and NQR did the OMOP to Phenopackets schema mapping. GD and PR did the generation pipeline of the synthetic OMOP database. AS created the SQL script and jointly with JJ they created the OMOP exporter code. GD created a similare exporter in python that leverages the same SQL instructions. RK, VT, JJ, VS and NQR did the modelling of semantic Phenopackets for Semantic Web based federated health infrastructures. PA and RK did the OMOP to semantic Phenopackets pipeline. RK, CF, RG, MK, LJC and NQR did the FAIR Machine Learning approaches and modelling discussion. NQR and TC drafted the introduction and the discussion sections, and NQR polished a final version of the manuscript. All authors reviewed and approved the final version of the manuscript.
 
```

Comment:
Hi @sasurfer, I may miss it, but is this explained in the manuscript? if yes, could you point me at the paragraph? if not could you add it? Thanks!

Reply:
The paragraph is called OMOP to Phenopackets in Python. It shortly describes a code in python that does the same thing the exporter does. Takes the data from OMOP with sql calls and writes phenopackets using the mapping made by the others and the python phenopackets library
Now I see a typo in what I wrote: similar not similare.

Reply:
Ah thank you for clarifying it. If both codes do exactly the same, but one is written in Python and the other in Java do you agree that it is a good idea to change the headings to make this clear? I propose to change the heading of 'OMOP to Phenopackets in Python' to 'Exporting patient-level phenopackets from an OMOP database in Python' or something like that, and the same for OMOP exporther heading: 'OMOP Exporter - Exporting patient-level phenopackets from an OMOP database in Java'

Reply:
They do the same thing but if I recall correctly the exporter does it with a web app while I did it with a python script.
I agree on giving the similar heading. Do you want me to add it or you prefer to add it yourself?

Reply:
I will. Thanks!